src/Time/Time.js

```diff
@@ -104,86 +116,92 @@ class Time extends Component {
+                        <label className='fd-time__slider-label'>{localizedText.secondsLabel}</label>
+                        <TimeItem
+                            active={active}
+                            defaultValue={1}
+                            disabled={disabled}
+                            disableStyles={disableStyles}
+                            downButtonProps={secondsDownButtonProps}
+                            format12Hours={format12Hours}
+                            inputProps={secondsInputProps}
+                            localizedText={{ buttonUp: localizedText.secondsUp, buttonDown: localizedText.secondsDown }}
+                            max={'60'}
+                            name='second'
+                            time={time}
+                            upButtonProps={secondsUpButtonProps}
+                            updateTime={this.updateTime}
+                            value={time.second} />
+                    </div>
                 ) : (
                     ''
                 )}
                 {/* Meridiem */}
                 {format12Hours ? (
-                    <TimeItem
-                        disabled={disabled}
-                        disableStyles={disableStyles}
-                        downButtonProps={meridiemDownButtonProps}
-                        inputProps={meridiemInputProps}
-                        localizedText={localizedText}
-                        max={'1'}
-                        name='meridiem'
-                        spinners={spinners}
-                        time={this.state.time}
-                        type={'Period'}
-                        upButtonProps={meridiemUpButtonProps}
-                        updateTime={this.updateTime}
-                        value={this.CLOCK[this.state.time.meridiem]} />
+                    <div className='fd-time__col' onClick={() => this.updateActiveColumn('meridiem')}>
+                        <label className='fd-time__slider-label'>{localizedText.meridiemLabel}</label>
+                        <TimeItem
+                            active={active}
+                            disabled={disabled}
+                            disableStyles={disableStyles}
+                            downButtonProps={meridiemDownButtonProps}
+                            inputProps={meridiemInputProps}
+                            localizedText={{ buttonUp: localizedText.meridiemUp, buttonDown: localizedText.meridiemDown }}
+                            max={'1'}
+                            name='meridiem'
+                            time={this.state.time}
+                            upButtonProps={meridiemUpButtonProps}
+                            updateActiveColumn={this.updateActiveColumn}
+                            updateTime={this.updateTime}
+                            value={this.CLOCK[this.state.time.meridiem]} />
+                    </div>
```

Comment:
Not sure if its related to this change, but the AM/PM selection seems off:
![2020-05-21 16-20-21 gifcask 2020-05-21 16_24_13](https://user-images.githubusercontent.com/43764832/82615583-a20af380-9b7f-11ea-9143-031f2d71ac8c.gif)

Reply:
Thanks for catching, fixed